app/views/feeds/_form_main.html.erb

```diff
@@ -0,0 +1,203 @@
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :feed_link, value: feed.published_url, disabled: true %>
+    <%= form.label :feed_link %>
+  </div>
+
+  <%= link_to(feed.published_url, class: "prx-btn-help", target: :_blank) do %>
+    <span class="material-icons">link</span>
+  <% end %>
+</div>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :file_name %>
+    <%= form.label :file_name %>
+  </div>
+
+  <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.file_name") %>"><span class="material-icons">help</span></button>
+</div>
+
+<% unless feed.default? %>
```

Comment:
wondering if this chunk of fields should be abstracted into its own partial? or if it makes sense to stay here?

Reply:
May be worth extracting _something_... this file is getting a bit long.  Abstract out the more advanced sections?